diff --git a/include/kernel-version.mk b/include/kernel-version.mk
index 6060776..ee6f4da 100644
--- a/include/kernel-version.mk
+++ b/include/kernel-version.mk
@@ -2,9 +2,9 @@
 
 LINUX_RELEASE?=1
 
-LINUX_VERSION-3.18 = .23
+LINUX_VERSION-3.18 = .80
 
-LINUX_KERNEL_MD5SUM-3.18.23 = dc6d265ab38716be3676ac294b481ad8
+LINUX_KERNEL_MD5SUM-3.18.80 = 5f2bba0500b3650fdaa0bf6e06062056
 
 ifdef KERNEL_PATCHVER
   LINUX_VERSION:=$(KERNEL_PATCHVER)$(strip $(LINUX_VERSION-$(KERNEL_PATCHVER)))
