diff --git a/scripts/download.pl b/scripts/download.pl
index cd68a7b..42df738 100755
--- a/scripts/download.pl
+++ b/scripts/download.pl
@@ -173,8 +173,8 @@ foreach my $mirror (@ARGV) {
 			push @extra, "$extra[0]/longterm/v$1";
 		}		
 		foreach my $dir (@extra) {
-			push @mirrors, "ftp://ftp.all.kernel.org/pub/$dir";
-			push @mirrors, "http://ftp.all.kernel.org/pub/$dir";
+			push @mirrors, "https://cdn.kernel.org/pub/$dir";
+			push @mirrors, "https://www.kernel.org/pub/$dir";
 		}
     } elsif ($mirror =~ /^\@GNOME\/(.+)$/) {
 		push @mirrors, "http://ftp.gnome.org/pub/GNOME/sources/$1";
